tests/unittest/_torch/modeling/test_modeling_exaone4.py

```
@@ -22,7 +22,7 @@ class Exaone4Config(PretrainedConfig):
     # TODO: Remove this once we have a proper config for Exaone4
     SKIP_EXAONE4_HF_ACCURACY_TEST = True
 
-from _torch.helpers import create_mock_engine
+from _torch.helpers import create_mock_cuda_graph_runner
```

Ruff (0.12.2)

Module level import not at top of file

(E402)